pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -122,9 +123,7 @@ rustPlatform.buildRustPackage rec {
     };
     terminfo = runCommand "wezterm-terminfo"
       {
-        nativeBuildInputs = [
-          ncurses
-        ];
+        nativeBuildInputs = [ ncurses ];
```

Comment:
I have put it into an extra commit together with some meta cleanup since I want to keep it. Diff size shouldn't be a problem when reviewing commit by commit.